docs/DeveloperGuide.md

```diff
@@ -198,6 +198,37 @@ Step 6. The model is updated accordingly through `ModelManager`.
 
 Step 7. A `CommandResult` is produced based on whether the execution was a success or not and returned to the `LogicManager`.
 
+
+### Finding Projects Feature
+
+The displayed list of `project`s can be filtered by name (specifically, the keywords that the project name contains).
```

Comment:
Should this and some of the other `project` in the documentation be `Project` instead, since they are classes?

Reply:
Yup, I have done so.